addons/mail/static/src/models/message_reaction_group_partner.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'MessageReactionGroupPartner',
+    fields: {
+        messageContextViewOwner: one('MessageContextMenu', {
+            identifying: true,
+            inverse: 'messageContextReactionPartners',
+        }),
+        messageReactionGroup: one('MessageReactionGroup', {
+            identifying: true,
+            inverse: 'messageContextReactionPartners'
+        }),
+        messageViewOwner: one('MessageView', {
+            related : 'messageContextViewOwner.messageViewOwner'
+        }),
```

Comment:
remove this `messageViewOwner` field (not a good field to introduce, leaks technical details of modelling that is not that desirable).